simics-rs/simics-build-utils/src/lib.rs

```diff
@@ -60,68 +37,224 @@ where
 /// SIMICS version of the low level bindings. This is not needed for all consumers of the
 /// API, but is useful for consumers which need to remain compatible with a wide range of
 /// SIMICS base versions.
-pub fn emit_cfg_directives() -> anyhow::Result<()> {
+///
+/// Changelog:
```

Comment:
I think you want to put this as a separate Markdown file in the docs folder so that it's easier to find/read.

Reply:
Good idea, moved to simics-rs/docs.